dbt/adapters/redshift/impl.py

```diff
@@ -115,6 +116,34 @@ def valid_incremental_strategies(self):
     def timestamp_add_sql(self, add_to: str, number: int = 1, interval: str = "hour") -> str:
         return f"{add_to} + interval '{number} {interval}'"
 
+    def _get_cursor(self):
+        return self.connections.get_thread_connection().handle.cursor()
+
+    def get_columns_in_relation(self, relation):
+        cursor = self._get_cursor()
+        results = []
+        if relation.identifier:
```

Comment:
yeah I think you're right, I added this bit because I just wanted to cover that scenario but I doubt there is a real use case at all for getting all columns in a schema